guide/en/how-to-open-up-data/from-identification-to-rawification-what-does-opening-government-data-means

```diff
@@ -0,0 +1,53 @@
+---
+title: From Identification to "Rawification": What Does Opening Government Data Mean?
+
+authors:
+ - *Jérôme Denis* 
+Associate Professor
+I3 (CNRS UMR 9217)
+Telecom ParisTech
+jerome.denis/at/telecom-paristech.fr
+- *Samuel Goëta*
+PhD Candidate, Telecom ParisTech
+Board member, Open Knowledge France
+samuel.goeta/at/telecom-paristech.fr
+
+---
+
+###Introduction
+Open government data has been at the center of several transparency policies and has been established as a legal obligation in many countries. Debates and discussions about open data have mainly focused on its opportunities and its capacity to develop transparency and innovation. Yet, how government data are concretely “opened” remain largely overlooked, as if data was, in its very nature, an easy available and shareable commodity. Is it really so simple?
+Rather, an exploration of the backrooms of open government data projects in several French administrations [1] shows that openness, far from being an intrinsic property of data, is the result of a specific work. Three main kinds of operations can be identified, which punctuate such work and lead to the production of open government data: identification, extraction and "rawification". They all show that data is not an available commodity within administrations that would be naturally ready for its public diffusion, mechanically producing transparency.
+
```

Comment:
Can we add here to whom this data is useful? I think that if we will add a line or two that will explain that this data is beneficial for civil society and why they should read about the back rooms of governments, it will go with the narrative of the whole book.